docs/docs/how-to/setup-intant-on-windows-wsl.mdx

```diff
@@ -0,0 +1,89 @@
+---
+id: setup-instant-on-windows-wsl
+title: How to setup Instant OpenHIE on Windows with WSL
+sidebar_label: Setup instant on Windows
+keywords:
+  - Instant OpenHIE
+  - How to
+  - Windows
+  - WSL
+description: How to setup Instant OpenHIE for development on Windows with WSL
+---
+
+:::info
+The Instant OpenHIE architecture, codebase, and documentation are under active development and are subject to change. While we encourage adoption and extension of the Instant OpenHIE framework, we do not consider this ready for production use at this stage.
+:::
+
+## 1. Install WSL
+Follow the steps on the Microsoft website to install the WSL feature for Windows:
+- [Windows Subsystem for Linux Installation Guide for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+## 2. Linux Distribution Setup
+
+If you haven't already done this as part of the step 1 above, you will need to create a user account and password for the Linux Distribution, the following link can assist with that:
+
+- [Create a user account and password for your new Linux distribution](https://docs.microsoft.com/en-us/windows/wsl/user-support)
+
+When using WSL 2 it is better to store files in the Linux subsystem file system rather than in the Windows file system for performance reasons. The easiest way to do this is to simply clone repositories and other files to the $HOME (~) directory on the Linux subsystem.
+
+## 3. Git Setup
+
+Many Linux distributions come with git out of the box, but if required, the following commands will install git:
+
+```sh
+sudo apt-get update
+```
+```sh
+sudo apt-get install git
+```
+
+## 4. Docker Desktop Setup
+
+Download and install Docker Desktop from:
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+## 5. Source Code & Development Environment Setup
+
+Download and install VS Code from:
+
+- [Download Visual Studio Code](https://code.visualstudio.com/download)
+
+Install the `Remote - WSL` VS Code extension for ease of working with your newly installed WSL 2 environment.
+
+Reload VS Code and open the terminal (default shortcut is Ctrl + ~), you should see the bash terminal come up and have an indication of the linux distribution that's running on the bottom left of the VS Code window. e.g.
+
+> WSL: Ubuntu-20.04
+
+Install npm and yarn
+
+```sh
+sudo apt-get install npm
+```
+
+```sh
+npm install yarn
+```
+
+Clone the Instant OpenHIE repository into the $HOME directory (or any sub-directory of $HOME).
+
+For HTTPS:
+```sh
+git clone https://github.com/openhie/instant.git
+```
+
+For SSH:
+```sh
+git clone git@github.com:openhie/instant.git
+```
+
+> Tip: Do not clone the repository or store any important files in the `/mnt/wsl` directory as this directory gets cleared every time the Windows `LxssManager` service gets restarted (every time you restart Windows).
+
+Build the Docker images using:
+
+```sh
+yarn docker:instant build
+```
+
```

Comment:
Could you please add in the start up command for core. And perhaps an OpenHIM Console check

```
yarn docker:instant init core
```

Reply:
@MattyJ007  I've added the start up command, could you please clarify what you mean with "an OpenHIM Console check"?

Reply:
Lols, sorry  - I mean just go to http://localhost:9000 and login to check that the OpenHIM has spun up properly.